class/defaults.yml

```
@@ -7,3 +7,7 @@ parameters:
     apiServerSpec:
       audit:
         profile: 'Default'
+
+    monitoring:
+      enabled: true
+      instance: ''
```

If we prefer `null` as the sentinel value, we should be consistent with it.

```suggestion
      instance: null
```